### PR TITLE
enh: add recipe for etelemetry

### PR DESCRIPTION
**Pull request by satra**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by satra** [comment]
@mgxd - could you please comment here that you will be a maintainer and check the box in the original checklist

**Comment by mgxd** [comment]
I can be a maintainer.
I can't check the OP but feel free to do it for me.

**Comment by satra** [comment]
@conda-forge-admin - is there a way to retrigger the build, according to staged-recipes, it seems to have been abandoned

**Comment by satra** [comment]
@conda-forge-bot help

**Comment by satra** [comment]
@conda-forge/python - how do i retrigger a build?

**Comment by effigies** [comment]
@satra Maybe close and re-open? Or just squash and force push.

**Comment by effigies** [comment]
@conda-forge/staged-recipes This is ready to go. This is a pure Python package.

File: recipes/etelemetry/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - requests
```
Comment (chrisburr) [inline recipes/etelemetry/meta.yaml]: This is probably only needed in `run`
```suggestion
```

File: recipes/etelemetry/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - python
+    - requests
+
+test:
+  imports:
+    - etelemetry
+
+about:
+  home: https://github.com/mgxd/etelemetry-client
+  license: Apache 2.0
```
Comment (chrisburr) [inline recipes/etelemetry/meta.yaml]:
```suggestion
  license: Apache-2.0
```

**Comment by satra** [comment]
@effigies - i also added you as a maintainer. could you please acknowledge that role?

**Comment by effigies** [comment]
Sure, I can be a maintainer.

**Pull request merged**